staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -129,6 +140,11 @@ type AuditWebhookOptions struct {
 	GroupVersionString string
 }
 
+type AuditDynamicOptions struct {
+	// Enabled tells whether the dynamic audit capability is enabled.
```

Comment:
Do we need this? The feature gate should be sufficient for now, the question is whether people will want to be able to disable it after the feature goes to GA.

Reply:
yeah this could be removed now and added before GA, I'll do that for simplicity sake

Reply:
feature gates are not intended to be long term enablement/disablement options. if the feature will need an option to be enabled/disabled as the feature graduates, it should have an explicit option for it